CHANGELOG.md

```diff
@@ -1,28 +1,52 @@
 Note: This is in reverse chronological order, so newer entries are added to the top.
 
 
+Swift 5.3.1
+-----------
+* [#2937]
+  
+  * Improvements
+    
+    `Package` manifests can now have any combination of leading whitespace characters. This allows more flexibility in formatting the manifests.
+    
+    [SR-13566] The Swift tools version specification in each manifest file now accepts any combination of _horizontal_ whitespace characters surrounding `swift-tools-version`, if and only if the specified version > 5.3. For example, `//swift-tools-version:	5.3.1` and `//		 swift-tools-version: 5.4` are valid.
+  
+    All [Unicode line terminators](https://www.unicode.org/reports/tr14/) are now recognised in `Package` manifests. This ensures correctness in parsing manifests that are edited and/or built on many non-Unix-like platforms that use ASCII or Unicode encodings. 
+  
+  * API Removal
+  
+    `ToolsVersionLoader.Error.malformedToolsVersion(specifier: String, currentToolsVersion: ToolsVersion)` is replaced by `ToolsVersionLoader.Error.malformedToolsVersionSpecification(_ malformation: ToolsVersionSpecificationMalformation)`.
+    
+    `ToolsVersionLoader.split(_ bytes: ByteString) -> (versionSpecifier: String?, rest: [UInt8])` and `ToolsVersionLoader.regex` are together replaced by `ToolsVersionLoader.split(_ manifest: String) -> ManifestComponents`.
+  
+  * Source Breakages for Swift Packages
+    
+    The package manager now throws an error if a manifest file contains invalid UTF-8 byte sequences.
+    
+    The package manager no longer silently falls back to using Swift 3.1 as the lowest supported version. Instead, a descriptive error is thrown for each misspelling or malformation in the manifest file.
+
 Swift 4.2
 ---------
 
-* [SE-209](https://github.com/apple/swift-evolution/blob/master/proposals/0209-package-manager-swift-lang-version-update.md)
```

Comment:
I hadn't really noticed these changes before, but what was the reason to remove the URL here?  That would seem to make it a bit harder to navigate to these proposals.